README.md

```diff
@@ -1,291 +1,124 @@
 <a name="readme-top"></a>
```

Comment:
_note: Since there is not PR in the frontend repo for me to review, I'll leave my comments here._

# Frontend review

- [ ]  Please, on App.js, kindly refactor the main"APP" functional component to ES6 arrow function, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered a non-best practice.

That way, your code will meet the latest standards. 💪

- [ ] I wasn't able to reserve a house with the proper dates, because there is no clear way to know the time and date format, it's just a box for text:


![image](https://github.com/Kgomotso196/House-Booking-Backend/assets/57421823/0c14c9c1-7980-4c16-9738-e3ea1517dec0)

![image](https://github.com/Kgomotso196/House-Booking-Backend/assets/57421823/5b52627e-408f-43be-9929-a1b486289b35)

Kindly, add some kind of date picker or some clarification on the time/date format so your times and dates are more consistent. 👍 

- [ ] The mobile view styling is questionable:

![image](https://github.com/Kgomotso196/House-Booking-Backend/assets/57421823/6ee8e985-c2a4-441a-8ede-fcbeb5e98077)

![image](https://github.com/Kgomotso196/House-Booking-Backend/assets/57421823/9df8c92f-3e85-4892-a600-6a8fabb4d92f)

![image](https://github.com/Kgomotso196/House-Booking-Backend/assets/57421823/e052f621-6bad-4df3-a526-ed98ebee9734)

Kindly, touch up your styling a little more so the elements on the mobile view fit the screen, and don't overlap/overflow each other. 👍 

That way, it will look professional.

- [ ] The desktop  view styling is questionable:

![image](https://github.com/Kgomotso196/House-Booking-Backend/assets/57421823/1084504f-c3ed-4d46-bedb-8c60aa19dd9d)

![image](https://github.com/Kgomotso196/House-Booking-Backend/assets/57421823/2870e6c8-9954-4d57-bb3e-4034665e3def)

Kindly, touch up your styling a little more so the elements on the mobile view fit the screen, and don't overlap/overflow each other. 👍 

That way, it will look professional.

_tip: most problems are because the navbar gets in the way._